odxtools/communicationparameter.py

```diff
@@ -48,9 +48,12 @@ def _read_complex_value_from_odx(et_element):
 
 def read_communication_param_ref_from_odx(et_element):
     id_ref = et_element.get("ID-REF")
-
+    # ODX standard v2.0.0 defined only VALUE
+    # ODX standard v2.0.1 decided to break things and change it to choice between SIMPLE-VALUE and COMPLEX-VALUE
```

Comment:
the ODX standard strictly adheres to the semversioning scheme ;)

Reply:
Wait until you see their "List of Known Issues" document.